services/symfony/symfony-insight-base.js

```diff
@@ -40,49 +39,41 @@ const keywords = ['sensiolabs', 'sensio']
 
 const gradeColors = {
   none: 'red',
-  bronze: '#C88F6A',
-  silver: '#C0C0C0',
-  gold: '#EBC760',
-  platinum: '#E5E4E2',
+  bronze: '#c88f6a',
+  silver: '#c0c0c0',
+  gold: '#ebc760',
+  platinum: '#e5e4e2',
 }
 
 class SymfonyInsightBase extends BaseXmlService {
   static get category() {
     return 'analysis'
   }
 
+  static get auth() {
+    return {
+      userKey: 'sl_insight_userUuid',
```

Comment:
just noticed the capital `U` in these keys, whereas the vast majority of others are all lowercase with underscores. Should the key checks in the auth helpers be case sensitive?

Reply:
All this stuff passes through the Joi schema in server.js so if we were gonna be lenient we’d need to do so there. As is people get loud failures when things are wrong, though a case error is admittedly very picky. I’d be fine if we updated the case in a later PR, don’t feel strongly one way or another.

Reply:
SGTM